src/com/homework/MyLinkedList.java

```
@@ -0,0 +1,79 @@
+package com.homework;
+
+public class MyLinkedList<E> implements List<E> {
+    private Node root;
+    private int countElements = 0;
+
+    @Override
+    public E get(int index) {
+        Node node;
+        if (index < 0 || index >= countElements) {
+            return null;
```

Null could be the value of the Node. Better to throw a custom exception.

---

The same.